docs/tabpy-tools.md

```diff
@@ -136,6 +137,102 @@ client.remove('WillItDefault')
 
 ```
 
+## Predeployed Functions
+
+To setup models download the latest version of TabPy and follow the instructions
+to install and start up your server. Once your server is running, navigate to the
+models directory and run setup.py.  If your TabPy server is running on the default
+port (9004), you do not need to specify a port when launching the script. If your
+server is running on a port other than 9004 you can specify a different port in
+the command line like so
+
+```sh
+
+python setup.py 4047
+
+```
+
+The setup file will install all of the necessary dependencies `(sklearn, nltk,
+textblob, pandas, & numpy)` and deploy all of the prebuilt models located in
+`./models/scripts`. For every model that is successfully deployed a message
+will be printed to the console
+
+```sh
+"Successfully deployed PCA"
+```
+
+If you would like to deploy additional models using the deploy script, you can
+copy any python file to the `./models/scripts` directory and modify setup.py to
+include all necessary packages when installing dependencies or alternatively install
+all the required dependencies manually.
+
+### Principal Component Analysis (PCA)
+
+[Principal component analysis](https://en.wikipedia.org/wiki/Principal_component_analysis)
+is a statistical technique which extracts new, linearly uncorrelated,
+variables out of a dataset which capture the maximum variance in the
+data. In this way, `PCA` can be used to reduce the number of variables
+in a high dimensional dataset, a process that is called dimensionality
+reduction. The first principal component captures the largest amount of
+variance, while the second captures the largest portion of the remaining
+variance while remaining orthogonal to the first and so on. This allows the
+reduction of the number of dimensions while maintaining as much of the
+information from the original data as possible. `PCA` is useful in
+exploratory data analysis because complex relationships can be visualized
+in a 2D scatter plot of the first few principal components.
+
+TabPy’s implementation of `PCA` uses the scikit-learn
+[decomposition.PCA](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html)
+algorithm, which is further documented
+[here](https://scikit-learn.org/stable/modules/decomposition.html#pca).
+It requires the selected component to be > 0 and <= number of variables you
```

Comment:
Say something like:

In the script, after the function name 'PCA', a specific principal component to return must be selected. This integer input should be > 0 and <= number of variables you pass into the function.